osu.Game/Rulesets/Objects/UnmanagedHitObjectEntry.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Rulesets.Objects.Drawables;
+
+namespace osu.Game.Rulesets.Objects
+{
+    internal class UnmanagedHitObjectEntry : HitObjectLifetimeEntry
+    {
+        public readonly DrawableHitObject DrawableHitObject;
+
+        public UnmanagedHitObjectEntry(HitObject hitObject, DrawableHitObject drawableHitObject)
+            : base(hitObject)
+        {
+            DrawableHitObject = drawableHitObject;
+            LifetimeStart = DrawableHitObject.LifetimeStart;
+            LifetimeEnd = DrawableHitObject.LifetimeEnd;
+        }
```

Comment:
This should really only accept the DHO and read the hitobject through it, no? Accepting the hitobject alongside the DHO like this is error-prone in this context.

Class also needs xmldoc as to what "unmanaged" means for future travelers.

Reply:
`DHO.HitObject` cannot be obtained here because the entry is not set.